tests/functional/testplan/runnable/interactive/test_interactive.py

```diff
@@ -535,7 +542,6 @@ def add_second_client_after_environment_started():
                     'server': 'STOPPED'})
 
 
-@pytest.mark.skipif(six.PY3, reason='Reload test is unstable on python 3')
```

Comment:
Do we know why it was unstable, and how this PR fixes it?

Reply:
It was unstable because it made some assumptions that are only valid on Python 2. It works with Python 3 fine now.